diff --git a/src/vs/workbench/contrib/welcomeGettingStarted/common/gettingStartedContent.ts b/src/vs/workbench/contrib/welcomeGettingStarted/common/gettingStartedContent.ts
index 168a3d01..775f4157 100644
--- a/src/vs/workbench/contrib/welcomeGettingStarted/common/gettingStartedContent.ts
+++ b/src/vs/workbench/contrib/welcomeGettingStarted/common/gettingStartedContent.ts
@@ -190,13 +190,13 @@ export const startEntries: GettingStartedStartEntryContent = [
 	},
 	{
 		id: 'topLevelRemoteOpen',
-		title: localize('gettingStarted.topLevelRemoteOpen.title', "Connect to..."),
-		description: localize('gettingStarted.topLevelRemoteOpen.description', "Connect to remote development workspaces."),
+		title: localize('gettingStarted.topLevelRemoteOpen.title', "New Workspace with Wizard"),
+		description: localize('gettingStarted.topLevelRemoteOpen.description', "Scaffold a new workspace by language, without typing a path."),
 		when: '!isWeb',
-		icon: Codicon.remote,
+		icon: Codicon.newFolder,
 		content: {
 			type: 'startEntry',
-			command: 'command:workbench.action.remote.showMenu',
+			command: 'command:codepad.newWorkspaceWizard',
 		}
 	},
 ];
